pkg/auth/tokens/issuer_factory.go

```diff
@@ -68,7 +68,7 @@ func (f *issuerFactory) createClaims(sourceID string, roxClaims RoxClaims) *Clai
 }
 
 func (f *issuerFactory) encode(claims *Claims) (string, error) {
-	return f.builder.Claims(&claims.Claims).Claims(&claims.RoxClaims).Claims(translateExtra(claims.Extra)).CompactSerialize()
+	return f.builder.Claims(&claims.Claims).Claims(&claims.RoxClaims).Claims(translateExtra(claims.Extra)).Serialize()
```

Comment:
Is this essentially the same as before?

Reply:
A shallow dive in the two versions suggests so:
The builder calls `CompactSerialize` on go-jose JSONWebSignature or JSONWebEncoding objects it built for the purpose.